src/common/slack.ts

```diff
@@ -135,16 +137,17 @@ export const notifyNewVordrPost = async (
     text: 'New post prevented by vordr',
     attachments: [
       {
-        title: post.title!,
+        title: sharedPost ? (post.title ?? sharedPost.title) : post.title,
```

Comment:
would it be more simple to say `post.title ?? sharedPost?.title`?

Reply:
Probably, I did these changes without running it locally. Could try it like that
```suggestion
        title: post.title ?? sharedPost?.title,
```

Reply:
You want title and if no title try shared title, else its empty either way? @omBratteng

Reply:
Yeah